### PR TITLE
feat: add common error module for socket

### PR DESCRIPTION
**Pull request by shuaijinchao**

### CODE REVIEW

File: src/runner/socket/error.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    A6ErrCode.Code.CONF_TOKEN_NOT_FOUND,
+    A6ErrCode.Code.BAD_REQUEST,
+    A6ErrCode.Code.SERVICE_UNAVAILABLE,
+]
+
+
+class New(object):
+    def __init__(self, code: int, message: str, data: bytes = b'', ty: int = 0):
+        if data is None:
+            data = {}
```
Comment (tao12345666333) [inline src/runner/socket/error.py:33]: Why set to an empty dict？

Comment (shuaijinchao) [reply src/runner/socket/error.py]: Thanks for review, this is a redundant piece of code and has been deleted.

**Review by spacewander — APPROVED** [review]

File: src/runner/socket/error.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+RUNNER_ERROR_CODE = 500
+
+errorCodes = [
+    A6ErrCode.Code.CONF_TOKEN_NOT_FOUND,
+    A6ErrCode.Code.BAD_REQUEST,
+    A6ErrCode.Code.SERVICE_UNAVAILABLE,
+]
+
+
+class New(object):
```
Comment (spacewander) [inline src/runner/socket/error.py:30]: What about using NamedTuple?

File: src/runner/socket/error.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import annotations
+from a6pluginproto.Err import Code as A6ErrCode
+
+RUNNER_SUCCESS_CODE = 200
+RUNNER_ERROR_CODE = 500
```
Comment (spacewander) [inline src/runner/socket/error.py:21]: Need some tests to show how this error code is used

**Comment by spacewander** [comment]
Please submit this PR into https://github.com/apache/apisix-python-plugin-runner/pull/8
See my comment in https://github.com/apache/apisix-python-plugin-runner/pull/8#discussion_r684563012

**Pull request closed without merging**